### PR TITLE
User Guide fixes

### PR DESCRIPTION
**Pull request by CyrilleB79**
### Link to issue number:

Fix-up of #15065
### Summary of the issue:

Various issues found in documentation while reviewing translation:

1. Flattened review:
> For example, if you move to the next object in this flattened view and the current object contains other objects, NVDA will automatically move to the first object that contains it.
Should be "that it contains", not "that contains it"!
Cc @leonardder  for approval

2. Excel UIA option not at the correct place in the documentation with respect to its position in the GUI's Advanced settings panel.

3. Searching for add-ons
> -Type a keyword or two for the kind of add-on you're looking for, then ``tab`` back to the list of add-ons.
Actually, we tab forward, not back.

4. Add-on Store spelling: Let's take care to use an unique writing so that people get immediately used:
So "Add-on Store", not "Add-ons Store".

5. Key name formatting:
Key name formatting should always be used for single keys, not for a whole list of keys including separators or parenthesis that do not belong to the name of a key as done for some braille keys.

### Description of user facing changes

Fixed all this.
### Description of development approach
Modified documentation.

### Note:

I was also planning to rewrite the paragraph "Move system caret when routing review cursor" which I find a bit hard to understand; however not sure to have the time for now.
Cc @XLTechie  if you are interested and for your opinion on this paragraph.
Also cc @burmancomp  and @leonardder for information and for your opinion.

### Testing strategy:
* Review
* Documentation checked again when translating
### Known issues with pull request:
None
### Change log entries:
None
### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
- [x] Security precautions taken.

### CODE REVIEW

**Comment by CyrilleB79** [comment]
Cc @seanbudd:
In addition to @Qchristensen's review, you may want to have a look to have points 4 (Add-on Store spelling) and 5 (key name formatting) so that you keep them in mind when reviewing future PRs, since you sometimes review documentation updates too.

**Comment by seanbudd** [comment]
I don't see any relevant changes in regards to point 4, could you explain why it was raised here?
In regards to point 5, I'm personally unfussed if separators like parenthesis and commas are included in the code formatting

**Review by seanbudd — DISMISSED** [review]

**Comment by seanbudd** [comment]
I see that there is one usage of "Add-ons store" in the first line of AddonStoreInstalling, was that meant to be changed in this PR?

**Review by Qchristensen — DISMISSED** [review]
Looks good, thanks!

File: user_docs/en/userGuide.t2t [hunk 1]
```diff
@@ -2232,6 +2232,14 @@ This setting contains the following values:
 - Always: where ever UI automation is available in Microsoft word (no matter how complete).
 -
 
+==== Use UI automation to access Microsoft  Excel spreadsheet controls when available ====[UseUiaForExcel]
+When this option is enabled, NVDA will try to use the Microsoft UI Automation accessibility API in order to fetch information from Microsoft Excel Spreadsheet controls.
+This is an experimental feature, and some features of Microsoft Excel may not be available in this mode.
+For instance, NVDA's Elements List for listing formulas and comments, and Browse mode quick navigation to jump to form fields on a spreadsheet features are not available.
+However, for basic spreadsheet navigating / editing, this option may provide a vast performance improvement.
+We still do not recommend that  the majority of users turn this on by default, though we do welcome users of Microsoft Excel  build 16.0.13522.10000 or higher to test this feature and provide feedback.
+Microsoft Excel's UI automation implementation is ever changing, and  versions of Microsoft Office older than 16.0.13522.10000 may not expose enough information for this option to be of any use.
```
Comment (seanbudd) [inline user_docs/en/userGuide.t2t:2241]: Fix up double space

```suggestion
==== Use UI automation to access Microsoft Excel spreadsheet controls when available ====[UseUiaForExcel]
When this option is enabled, NVDA will try to use the Microsoft UI Automation accessibility API in order to fetch information from Microsoft Excel Spreadsheet controls.
This is an experimental feature, and some features of Microsoft Excel may not be available in this mode.
For instance, NVDA's Elements List for listing formulas and comments, and Browse mode quick navigation to jump to form fields on a spreadsheet features are not available.
However, for basic spreadsheet navigating / editing, this option may provide a vast performance improvement.
We still do not recommend that the majority of users turn this on by default, though we do welcome users of Microsoft Excel build 16.0.13522.10000 or higher to test this feature and provide feedback.
Microsoft Excel's UI automation implementation is ever changing, and versions of Microsoft Office older than 16.0.13522.10000 may not expose enough information for this option to be of any use.
```

**Comment by CyrilleB79** [comment]
> I see that there is one usage of "Add-ons store" in the first line of AddonStoreInstalling, was that meant to be changed in this PR?

Yes, that was it. I had forgotten to push it yesterday. Now fixed in 7f4b21d009f62c06aefd0b36308d04e3bc70641e.

**Comment by CyrilleB79** [comment]
> In regards to point 5, I'm personally unfussed if separators like parenthesis and commas are included in the code formatting

IMO it's better not to add them, for example:
When writing "Press ctrl+c and   ctrl+v", you will not use key name formatting for the word "and". The same way, if you write instead "Press ctrl+c, ctrl+v", the comma should not be formatted as key names. Semantically, "ctrl+c" and "ctrl+v" are two distinctive shortcuts separated by "and" or by a comma.

Of course, in a comma in a sequential shortcut such "alt, f, w" found in Word's ribbons, the comma belongs to the shortcut and should be formatted along with the whole shortcut formatting.

This formatting issue is not very important, so I won't submit a PR to fix them (if any) in the current version of the Guide. But checking this in newly added content could be a good practice IMO.

**Comment by XLTechie** [comment]
Agreed with @CyrilleB79 on this. If we're going to use code markers around 
shortcuts, then they should be used only around shortcuts, as indicating the 
actual "code" in question. Since the added punctuation is not part of the code, 
it shouldn't be included.

Though I also agree it is minor.

**Pull request merged**